docs/modules/commons/pages/vividus-steps.adoc

```diff
@@ -261,3 +261,85 @@ When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'`
 When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'` against `prod` and save result to scenario variable `prod-data`
 Then `${preprod-data}` is = `${prod-data}`
 ----
+
+=== Execute steps N times
+
+Step is designed to perform `$stepsToExecute` while counter matches `$comparisonRule`. The limit for the counter is set by `$limit` variable. On each iteration the counter is increased on `$increment` value, which is allowed to be either positive or negative. The `$seed` value is used as a starting point for iteration. Current iteration index is available within `$stepsToExecute` as `${iterationVariable}`.
+
+[source,gherkin]
+----
+When I execute steps while counter is $comparisonRule `$limit` with increment `$increment` starting from `$seed`:$stepsToExecute
+----
+
+Alias:
+[source,gherkin]
+----
+When I execute steps while counter is $comparisonRule '$limit' with increment '$increment' starting from '$seed':$stepsToExecute
+----
+
+* `$comparisonRule` - xref:parameters:comparison-rule.adoc[The comparison rule].
+* `$limit` - The counter limit.
+* `$increment` - The number to add to the counter on each iteration.
+* `$seed` - The initial counter value.
+* `$stepsToExecute` - The ExamplesTable with a single column containing the steps to execute.
+
+.Press the button four times
+[source,gherkin]
+----
+When I execute steps while counter is less than or equal to `10` with increment `3` starting from `1`:
+|step                                                            |
+|When I click on an element with the text 'CLICK ME FOUR TIMES!' |
+----
+
+=== Execute steps when a condition is true
+
+Steps designed to perform steps if condition is true.
+
+[source,gherkin]
+----
+When the condition `$condition` is true I do$stepsToExecute
+----
+
+Alias:
+[source,gherkin]
+----
+When the condition '$condition' is true I do$stepsToExecute
+----
+
+* `$condition` - The verifiable condition.
```

Comment:
Please add the description how is result of the condition checked: true vs anything to else